src/main/modules/recaptcha.js

```diff
@@ -0,0 +1,26 @@
+import $ from 'jquery'
+
+const ReCaptcha = ($container, index) => {
+  window[`reCaptchaCallbacks_${index}`] = submitCallback($container)
+  $('button.g-recaptcha', $container).attr(
+    'data-callback',
+    `reCaptchaCallbacks_${index}`
+  )
+}
+
+const submitCallback = $el => () => {
+  $el.submit()
+}
+
+const injectReCaptchaScript = () => {
```

Comment:
IMHO, you could inline that method

Reply:
I think that the code is easier to read with the name, so I'll leave that as it is.

Reply:
fair enough :)